code/modules/admin/verbs/BloomEdit.dm

```diff
@@ -0,0 +1,74 @@
+	data["glow_brightness_power"] = GLOB.configuration.lighting_effects.glow_brightness_power
+	data["glow_contrast_base"] = GLOB.configuration.lighting_effects.glow_contrast_base
+	data["glow_contrast_power"] = GLOB.configuration.lighting_effects.glow_contrast_power
+	data["exposure_brightness_base"] = GLOB.configuration.lighting_effects.exposure_brightness_base
+	data["exposure_brightness_power"] = GLOB.configuration.lighting_effects.exposure_brightness_power
+	data["exposure_contrast_base"] = GLOB.configuration.lighting_effects.exposure_contrast_base
+	data["exposure_contrast_power"] = GLOB.configuration.lighting_effects.exposure_contrast_power
+	return data
+
+/datum/bloom_edit/ui_act(action, list/params, datum/tgui/ui, datum/ui_state/state)
+	. = ..()
+	if(.)
+		return
+	switch(action)
+		if("glow_brightness_base")
+			GLOB.configuration.lighting_effects.glow_brightness_base = clamp(params["value"], -10, 10)
+		if("glow_brightness_power")
+			GLOB.configuration.lighting_effects.glow_brightness_power = clamp(params["value"], -10, 10)
+		if("glow_contrast_base")
+			GLOB.configuration.lighting_effects.glow_contrast_base = clamp(params["value"], -10, 10)
+		if("glow_contrast_power")
+			GLOB.configuration.lighting_effects.glow_contrast_power = clamp(params["value"], -10, 10)
+		if("exposure_brightness_base")
+			GLOB.configuration.lighting_effects.exposure_brightness_base = clamp(params["value"], -10, 10)
+		if("exposure_brightness_power")
+			GLOB.configuration.lighting_effects.exposure_brightness_power = clamp(params["value"], -10, 10)
+		if("exposure_contrast_base")
+			GLOB.configuration.lighting_effects.exposure_contrast_base = clamp(params["value"], -10, 10)
+		if("exposure_contrast_power")
+			GLOB.configuration.lighting_effects.exposure_contrast_power = clamp(params["value"], -10, 10)
+		if("default")
+			GLOB.configuration.lighting_effects.glow_brightness_base = initial(GLOB.configuration.lighting_effects.glow_brightness_base)
+			GLOB.configuration.lighting_effects.glow_brightness_power = initial(GLOB.configuration.lighting_effects.glow_brightness_power)
+			GLOB.configuration.lighting_effects.glow_contrast_base = initial(GLOB.configuration.lighting_effects.glow_contrast_base)
+			GLOB.configuration.lighting_effects.glow_contrast_power = initial(GLOB.configuration.lighting_effects.glow_contrast_power)
+			GLOB.configuration.lighting_effects.exposure_brightness_base = initial(GLOB.configuration.lighting_effects.exposure_brightness_base)
+			GLOB.configuration.lighting_effects.exposure_brightness_power = initial(GLOB.configuration.lighting_effects.exposure_brightness_power)
+			GLOB.configuration.lighting_effects.exposure_contrast_base = initial(GLOB.configuration.lighting_effects.exposure_contrast_base)
+			GLOB.configuration.lighting_effects.exposure_contrast_power = initial(GLOB.configuration.lighting_effects.exposure_contrast_power)
+		if("update_lamps") // todo: make this update all objects with glow
```

Comment:
Either resolve this todo, or make it explain why it's not straightforward to do.